<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1735</width>
    <height>1055</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>-390</x>
     <y>-100</y>
     <width>2411</width>
     <height>1331</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>420</y>
     <width>221</width>
     <height>331</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="glaza_in_right">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Глаза вправо</string>
    </property>
   </widget>
   <widget class="QPushButton" name="glaza_in_left">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Глаза влево</string>
    </property>
   </widget>
   <widget class="QPushButton" name="open_rot">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть рот </string>
    </property>
   </widget>
   <widget class="QPushButton" name="close_rot">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Закрыть рот</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>221</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>Выход</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exit</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>669</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>692</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
